config/locales/sufia.en.yml

```diff
@@ -443,3 +474,8 @@ en:
       admin_set:
         create: 'Save'
         update: 'Save'
+      sufia_permission_template:
```

Comment:
because the model is in the Sufia namespace. This is how rails finds a i18n key for a model in a namespace.